src/components/search/data/searchLoader.js

```diff
@@ -33,6 +33,15 @@ export default function makeSearchLoader(queryClient) {
 
     await Promise.all(searchData);
 
+    const academies = await queryClient.getQueryData(academiesListQuery.queryKey);
```

Comment:
nit: `queryClient.getQueryData` is a synchronous operation; shouldn't need to `await` it.